nilearn/surface/tests/test_surface.py

```diff
@@ -7,17 +7,13 @@
 
 from distutils.version import LooseVersion
 
+import nibabel as nb
+import numpy as np
 import pytest
 
 from numpy.testing import assert_array_equal, assert_array_almost_equal
-
-from nilearn._utils.testing import assert_raises_regex, assert_warns
-
-import numpy as np
```

Comment:
ts ts ts this should be in another PR ;-)

Reply:
I know!!! And yet...